src/control-manager.cpp

```diff
@@ -417,7 +433,7 @@ namespace dynamicgraph
           }
         }
         else
-          SEND_MSG("Cannot change control mode of joint "+JointUtil::get_name_from_id(jid)+
+          SEND_MSG("Cannot change control mode of joint "+m_robot->model().getJointName(jid)+
```

Comment:
Fix in commit bc5f539678009aa87d3eaa2a26e524133f5d03dd. But IMHO adding a remapping between name and id in a complex robotics application, in the long run, may just generate another set of confusion for newcomers...